roles/autotested/tasks/main.yml

```diff
@@ -107,19 +107,24 @@
   debug: var=docker_autotest
   when: docker_autotest is defined
 
-- name: Execute Docker autotest
-  shell: "export AUTOTEST_PATH={{ autotest_dest_dir }} && $AUTOTEST_PATH/client/autotest-local {{ autotest_docker_path }}/control"
-  args:
-    chdir: "{{ autotest_docker_path }}"
-  register: autotest_execution
-  async: 14401
-  poll: 60
-  when: docker_autotest.execute
-  # Any Autotest failures will eventually be reflected as missing results
-  ignore_errors: True
+- block:
+
+    - name: Execute Docker autotest
+      shell: "export AUTOTEST_PATH={{ autotest_dest_dir }} && $AUTOTEST_PATH/client/autotest-local {{ autotest_docker_path }}/control"
+      args:
+        chdir: "{{ autotest_docker_path }}"
+      register: autotest_execution
+      async: 14401
+      poll: 60
+      # Any Autotest failures will eventually be reflected as missing results
+      ignore_errors: True
 
-- include: produce_results.yml
-  when: autotest_execution is defined
+  always:
```

Comment:
works just like an ``except...finally`` in python, except it doesn't re-raise the error (in ansible)